ext/intl/tests/dateformat_format_relative.phpt

```diff
@@ -38,19 +38,19 @@ function printFormat(int $dateFormat, int $timeFormat, DateTimeImmutable $time)
 }
 
 function getToday(): DateTimeImmutable {
-    return new DateTimeImmutable();
+    return new DateTimeImmutable("now", new DateTimeZone("America/Los_Angeles"));
```

Comment:
I think you can just do `date_default_timezone_set('America/Los_Angeles');` at the top of the file.

Reply:
Thank you!
I changed in https://github.com/php/php-src/pull/12637/commits/f83481519f930dfea1a5a43465a81385c89e8889